lib/plugins/threaded-signing/plugin.c

```diff
@@ -30,32 +30,36 @@
 #include <glib.h>
 #include <stdlib.h>  // calloc, malloc, free
 #include <string.h>  // memcpy
+#include <unistd.h>
 
 #include "includes/signed_video_interfaces.h"
 #include "includes/signed_video_openssl.h"
 
-typedef enum {
-  THREADED_SIGNING_WAITS_FOR_HASH_TO_SIGN,
-  THREADED_SIGNING_HAS_HASH_TO_SIGN,
-  THREADED_SIGNING_SIGNS_HASH,
-  THREADED_SIGNING_HAS_SIGNATURE,
-  THREADED_SIGNING_ERROR,
-} threaded_signing_plugin_state;
+#define MAX_HASHES 60
```

Comment:
How about renaming to `MAX_BUFFER_LENGTH` since it is used both for input (where it indeed is hashes) and output (where it includes signature and more)